app/views/landing/index.html.erb

```diff
@@ -1,45 +1,15 @@
-<%= render partial: 'header' %>
+<%= render partial: 'new_header' %>     
 
```

Comment:
@andela-tfowotade Please remove extra space here and the trailing whitespace on the previous line